addons/im_livechat/static/src/components/discuss/discuss_tests.js

```diff
@@ -336,7 +401,7 @@ QUnit.test('livechats are sorted by last message date in the sidebar: most recen
     );
 });
 
-QUnit.test('livechats with no messages are sorted by creation date in the sidebar: most recent at the top', async function (assert) {
```

Comment:
Hello, changing this test is not ok. It must be possible (from website visitor) to create a new empty chat to visitor, and this chat must be visible.